pkgs/development/compilers/llvm/common/libc/default.nix

```diff
@@ -0,0 +1,94 @@
+{
+  lib,
+  stdenv,
+  llvm_meta,
+  src ? null,
+  monorepoSrc ? null,
+  version,
+  release_version,
+  runCommand,
+  python3,
+  python3Packages,
+  patches ? [ ],
+  cmake,
+  ninja,
+  isFullBuild ? true,
+  linuxHeaders,
+}:
+let
+  pname = "libc";
+
+  src' =
+    if monorepoSrc != null then
+      runCommand "${pname}-src-${version}" { } (''
+        mkdir -p "$out"
+        cp -r ${monorepoSrc}/cmake "$out"
+        cp -r ${monorepoSrc}/runtimes "$out"
+        cp -r ${monorepoSrc}/llvm "$out"
+        cp -r ${monorepoSrc}/${pname} "$out"
+      '')
+    else
+      src;
+
+  stdenv' =
+    if stdenv.cc.isClang then
+      stdenv.override {
+        cc = stdenv.cc.override {
+          nixSupport = stdenv.cc.nixSupport // {
+            cc-cflags = lib.remove "-lunwind" stdenv.cc.nixSupport.cc-cflags;
+          };
+        };
+      }
+    else
+      stdenv;
+in
+stdenv'.mkDerivation (finalAttrs: {
+  inherit pname version patches;
+
+  src = src';
+
+  sourceRoot =
+    if lib.versionOlder release_version "13" then null else "${finalAttrs.src.name}/runtimes";
+
+  nativeBuildInputs =
+    [
+      cmake
+      python3
+    ]
+    ++ (lib.optional (lib.versionAtLeast release_version "15") ninja)
+    ++ (lib.optional finalAttrs.passthru.isFullBuild python3Packages.pyyaml);
```

Comment:
It's strange not to just write isFullBuild here.  Otherwise, you're supporting changing it both via override and overrideAttrs, which isn't how packages in Nixpkgs usually work.

Reply:
Fixed